echo/testdata/golden/nginx.json

```diff
@@ -0,0 +1,4 @@
+{
+  "CVE-2021-3618": { "fixed_version": "1.21.0" },
+  "CVE-2023-44487": { "fixed_version": "1.25.2" }
+}
```

Comment:
Try to use same files.
`utils.Write` adds indents - https://github.com/orizerah/vuln-list/blob/main/echo/nginx.json